bosh-setup/azuredeploy.json

```diff
@@ -66,6 +66,7 @@
   },
   "variables": {
     "apiVersion": "2015-06-15",
```

Comment:
Got it. Thanks.
Will remove the var `apiVersion`, and specify `apiVersion` in each resource.